.github/workflows/dhis2-verify-app.yml

```diff
@@ -108,16 +108,35 @@ jobs:
 
     release:
         runs-on: ubuntu-latest
-        needs: [publish]
-        if: "github.event_name == 'push' && !contains(github.event.head_commit.message, '[skip ci]')"
+        needs: [build, lint, test] # add e2e if you use it
```

Comment:
there is no `test` action in this workflow, so will need to change this to depend on [build, lint] only